amida-tech/amida-auth-microservice
# Feature/orange 159 twilio integration

If this PR fixes a bug, you _must_ add test cases representative of the bug.
#### What's this PR do?
Adds phone number and twilio integration 
#### Related JIRA tickets:
https://jira.amida-tech.com/browse/ORANGE-159
#### How should this be manually tested?
Service will have to be re ran to update database
Twilio configuration needs to be added (check example env) 
Create user with only phone number (this format is the only accepted currently 3335557777)
Reset password
#### Any background context you want to provide?
Phone number validation will eventually need to be improved depending on the desire to use international numbers. 
#### Screenshots (if appropriate):
